Source/WTF/wtf/PlatformHave.h

```diff
@@ -838,15 +830,16 @@
 #define HAVE_APPLE_LOW_POWER_MODE_SUPPORT 1
 #endif
 
+// FIXME: Why no macCatalyst?
```

Comment:
I think that `PLATFORM(MAC)` includes macCatalyst. So this comment isn't correct, and the below is just `PLATFORM(COCOA)`. Also, we can probably remove `HAVE_LIBXSLT_FIX_FOR_RADAR_71864140` completely now, relying on the fix unconditionally.

Reply:
I'll remove it in a follow-up patch (I'm aiming for these cleanups to not have any behavior change).